stl/inc/yvals_core.h

```diff
@@ -751,6 +751,12 @@
     "Either use a Standard specialization or define _ENFORCE_FACET_SPECIALIZATIONS=0 " \
     "to suppress this diagnostic."
 
+#ifdef _ALLOW_RANDOM_DISTRIBUTION_CONST_OPERATOR

```

Comment:
I think we should put this in `<random>`, and `#undef` the macro at the end so it doesn't leak out of that header. We don't need it anywhere else, and localization would make me less nervous about using `_CONST`.